.github/workflows/luacheck.yml

```diff
@@ -0,0 +1,13 @@
+name: luacheck
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: apt
+      run: sudo apt-get install -y luarocks
+    - name: luacheck install
+      run: luarocks install --local luacheck
+    - name: luacheck run
+      run: $HOME/.luarocks/bin/luacheck ./
```

Comment:
Bridger uses tabs, not spaces, if that's allowed for YAML.

Reply:
Interestingly, they are not allowed: http://yaml.org/faq.html